allennlp/modules/seq2seq_encoders/multi_head_self_attention.py

```diff
@@ -142,10 +134,9 @@ def forward(self,  # pylint: disable=arguments-differ
         # which is equivalent to a weighted sum of the values with respect to
         # the attention distributions for each element in the num_heads * batch_size
         # dimension.
-        # shape (num_heads * batch_size, timesteps, values_dim)
+        # shape (num_heads * batch_size, timesteps, values_dim/num_heads)
         outputs = torch.bmm(attention, values_per_head)
```

Comment:
You could just actually use our `weighted_sum` here, which internally is `torch.bmm` for tensors with these dimensions.  Then you wouldn't need this long comment, and it'd be easier to read.